tests/hw-accel/nfd/internal/wait/wait.go

```diff
@@ -87,19 +91,31 @@ func ForNodeReadiness(apiClient *clients.Settings,
 }
 
 // ForPod check that all pods in namespace are in running state.
-func ForPod(apiClient *clients.Settings, nsname string) (bool, error) {
-	podList, err := pod.List(apiClient, nsname, metav1.ListOptions{})
+func ForPod(apiClient *clients.Settings, timeout time.Duration, nsname string) (bool, error) {
+	err := wait.PollUntilContextTimeout(
+		context.TODO(), 5*time.Second, timeout, true, func(ctx context.Context) (bool, error) {
+			pods, err := get.PodStatus(apiClient, hwaccelparams.NFDNamespace)
+			if err != nil {
+				return false, err
+			}
+
+			for _, pod := range pods {
+				if pod.State != "Running" {
```

Comment:
You could use the standard definition of the pod running, and not `Running` string. 

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L3329